cmd/get.go

```diff
@@ -27,11 +28,19 @@ import (
 )
 
 func get(cmd *cobra.Command, args []string) (err error) {
+	if len(args) == 0 || len(args) > 2 {
+		return errors.New("`get` requires 1 or 2 arguments")
```

Comment:
That *could* be interpreted as meaning it's OK to pass `dst` without `src`. I don't think that's likely though.